pages/create.js

```diff
@@ -0,0 +1,26 @@
+import ServiceForm from "@/components/ServiceForm";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { addService } from "@/lib/api";
+import { mutate } from "swr";
+
+export default function AddServicePage() {
+  const router = useRouter();
+
+  async function onSubmit(data) {
+    await addService({ ...data });
+    mutate(`api/services`);
+    router.push("/");
+  }
+
+  return (
+    <>
+      <ServiceForm onSubmit={onSubmit} />
+      <div>
+        <Link href="/">
+          <p>cancel</p>
+        </Link>
+      </div>
```

Comment:
I suggest to delete the `div` element here, as it does not serve any purpose here. Additionally, you do not need a `p` element inside the `Link` component. You can simply write the content of the `Link` inbetween the opening and closing tag.
```suggestion

        <Link href="/">
         cancel
        </Link>

```